Library/Homebrew/extend/ENV/std.rb

```diff
@@ -59,33 +52,17 @@ def setup_build_environment(formula = nil)
       gcc_formula = gcc_version_formula($&)
       append_path "PATH", gcc_formula.opt_bin.to_s
     end
+  end
+  alias_method :setup_generic_build_environment, :setup_build_environment
```

Comment:
I agree that this sounds better, but wouldn't it be safer to stick to a naming scheme where these aliased methods are *always* prefixed with `generic_`?